src/yggdrasil/search.go

```diff
@@ -24,7 +24,8 @@ import (
 
 // This defines the time after which we time out a search (so it can restart).
 const search_RETRY_TIME = 3 * time.Second
-const search_STEP_TIME = 100 * time.Millisecond
+const search_STEP_TIME = time.Second
```

Comment:
I should mention, this time is really arbitrary. I'm trying to error on the side of minimizing traffic by not sending packets too aggressively when things aren't timing out or reaching a dead end.